README.md

```diff
@@ -21,18 +21,21 @@ Configuration is done through environment variables. The following configuration
 
 The following configuration options are optional.
 
-| Environment Variable  | Description                                                                                                                             |
-| --------------------  | --------------------------------------------------------------------------------------------------------------------------------------- |
-| CALLER_KEY            | The caller key that WiseTime should provide with post time webhook calls. The connector does not authenticate Webhook calls if not set. |
-| TAG_UPSERT_PATH       | The WiseTime tag folder path to use for Jira tags. Defaults to `/Jira/` (trailing slash is required). Use `/` for root folder.          |
-| TAG_UPSERT_BATCH_SIZE | Number of tags to upsert at a time. A large batch size mitigates API call latency. Defaults to 500.                                     |
+| Environment Variable              | Description                                                                                                                             |
+| --------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| CALLER_KEY                        | The caller key that WiseTime should provide with post time webhook calls. The connector does not authenticate Webhook calls if not set. |
+| TAG_UPSERT_PATH                   | The WiseTime tag folder path to use for Jira tags. Defaults to `/Jira/` (trailing slash is required). Use `/` for root folder.          |
+| TAG_UPSERT_BATCH_SIZE             | Number of tags to upsert at a time. A large batch size mitigates API call latency. Defaults to 500.                                     |
+| ONLY_UPSERT_TAGS_FOR_PROJECT_KEYS | If set, the connector will only find new Jira issues for the configured Jira project keys.                                              | 
```

Comment:
The param name `ONLY_UPSERT_TAGS_FOR_PROJECT_KEYS` seems to be expecting a boolean value. How about `PROJECT_KEYS_FILTER` or  just `PROJECT_KEYS`. Then provide the expected format (e.g. CSV).

Reply:
I like `PROJECT_KEYS_FILTER`. Succinct and descriptive at the same time. Will change 👍 

Originally was meant to be only for tag upserts, but thinking through the name change it occurs to me that we should also filter on time posting. Consider two connectors, one for `DMS` project, and one for `WT`, both connected to the same Jira instance. If time is posted, we only want one connector to add it to Jira, or we'll get duplicate worklogs.

I'll make the necessary changes.